sale_project_fixed_price_task_completed_invoicing/README.md

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+Sale Project Fixed Price Task Completed Invoicing
+=================================================
+
+The main goal of this module is to add the possibility to link a sale.order.line
+to a project.task considering the delivery.
+Unless sale_timesheet, the quantity shipped won't be linked to a Timesheet nor
+to the time spent on the task. The price is fixed on the sale.order.line and it
+will be considered as shipped once the task is accomplished.
+
+Usage
+=====
+
+Create a product with product.type 'Service' and track_service 'Completed Task'.
```

Comment:
In this case I don't want to track hours nor to change the default behaviour that comes from sale_timesheet.